spec/requests/v1/songs_request_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+describe 'songs endpoints' do
+  describe 'GET /index' do
+    it 'returns JSON for all songs' do
+      songs = create_list(:song, 10)
+      get(songs_url, {}, accept_headers)
+      expect(response).to have_http_status :ok
+      expect(response).to match_response_schema :songs
+    end
```

Comment:
I usually put some blank lines in here between the setup & the expectations, just for readability.